server/sonar-server/src/main/java/org/sonar/server/ws/WebServiceEngine.java

```diff
@@ -60,11 +62,22 @@
 
   private final WebService.Context context;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this use of "WebService"; it is deprecated. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3ACallToDeprecatedMethod)